visualization/README.md

```diff
@@ -1,5 +1,3 @@
 # eolearn.visualization subpackage
 
 A collection of data visualization utilities.
-
-For more information on the package content, visit [readthedocs](https://eo-learn.readthedocs.io/en/latest/eolearn.visualization.html).
```

Comment:
forgot this one ;)

Reply:
Added only the link for the module content, since no tasks link exists.